portal-frontend/src/app/features/view-submission/lfng-review/lfng-review.component.scss

```diff
@@ -2,9 +2,31 @@
 @use '../../../../styles/colors';
 
 .warning {
+  align-items: center;
   background-color: rgba(colors.$accent-color-light, 0.5);
+  border-radius: rem(4);
+  display: flex;
   padding: rem(16);
   margin-bottom: rem(24);
+
+  mat-icon {
+    margin-right: 12px;
+  }
+}
+
+.comment-container {
```

Comment:
Don't use extends when adding both css classes is sufficient.

Reply:
Fixed!